plugins/rsyslog.yaml

```diff
@@ -1,21 +1,41 @@
 # Plugin Info
-version: 0.0.3
+version: 0.0.4
 title: Rsyslog
 description: Log parser for Rsyslog
 parameters:
+  - name: source
+    label: Log source
+    description: Use this field to specify where your logs are coming from. When choosing the 'file' option, the agent reads in logs from the log paths specified below. When choosing the 'tcp' or 'udp' options, the agent will listen on specified port.
+    type: enum
+    valid_values:
+      - file
+      - udp
+      - tcp
+    default: udp
   - name: listen_port
     label: Listen Port
     description: A port which the agent will listen for syslog messages
     type: int
     default: 514
-  - name: connection_type
-    label: Connection Type
-    description: The type of Rsyslog connection (`udp` or `tcp`)
+  - name: rsyslog_log_path
+    label: Rsyslog Log Path
+    description: Path to Rsyslog log file
+    type: string
```

Comment:
lets use strings her, in order to support multiple files, such as
- /var/log/secure
- /var/log/messages